src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__title {
+    color: $title-color-hover;
+  }
+
+  &__image {
+    margin-left: 3px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    margin: 36px 0 0;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-accent;
+  }
+
+  &__product-id {
+    margin: 4px 0 0;
+    color: $secondary;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```

Comment:
Remove redundant default value. Fix it everywhere

Reply:
✅